### PR TITLE
MDLSITE-5185 admin: Add mobile related information to registration

### PR DESCRIPTION
**Pull request by jleyva**

### CODE REVIEW

**Comment by mudrd8mz** [comment]
Thanks Juan for your work on this. Just a small reminder for the future, it will help to have the actual code change in a separate commit from other changes that only fix the code formatting guidelines.

**Comment by mudrd8mz** [comment]
As asked in the tracker, I will appreciate a new MDLSITE issue to track the deployment process of this change. Thanks in advance.

**Comment by marinaglancy** [comment]
I created https://tracker.moodle.org/browse/MDLSITE-5185 for it

File: db/install.xml [hunk 1]
```diff
@@ -54,6 +54,10 @@
         <FIELD NAME="fingerprint" TYPE="char" LENGTH="255" NOTNULL="false" SEQUENCE="false"/>
         <FIELD NAME="cool" TYPE="int" LENGTH="10" NOTNULL="false" DEFAULT="0" SEQUENCE="false"/>
         <FIELD NAME="cooldate" TYPE="int" LENGTH="10" NOTNULL="false" DEFAULT="0" SEQUENCE="false"/>
+        <FIELD NAME="mobileservicesenabled" TYPE="int" LENGTH="1" NOTNULL="false" DEFAULT="-1" SEQUENCE="false" COMMENT="Whether mobile services are enabled in the site, -1 if unknown"/>
+        <FIELD NAME="mobilenotificacionsenabled" TYPE="int" LENGTH="1" NOTNULL="false" DEFAULT="-1" SEQUENCE="false" COMMENT="Whether mobile notifications are enabled in the site, -1 if unknown"/>
```
Comment (scara) [inline db/install.xml]: Hi @jleyva,
typo in the field name: it looks like a bit of spanglish here :smile: .

HTH,
Matteo

**Comment by marinaglancy** [comment]
Thanks @scara , good catch, I created MDL-59770 to fix it in moodle core too

**Comment by mudrd8mz** [comment]
Closing this pull request without pulling. @jleyva please contact me for further details, thanks.

**Pull request closed without merging**